04-Intro_to_Vue/assignment/lesson-04-vue/src/components/GridPage.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="d-flex flex-column min-vh-100">
+    <!-- Navbar -->
+    <AppNavbar />
+
+    <!-- Main Content -->
+    <main class="container mt-4 flex-grow-1">
+      <h1>Species of Dinosaurs</h1>
+      <b-row>
+        <!-- Grid Item #1 -->
+        <b-col cols="12" sm="6" md="4" class="mb-4">
```

Comment:
These should be part of a `v-for` as they can render over data and return cards.

Reply:
For sure. Sometimes...well, you just gotta. :)

Reply:
This is definitely going to get added soon!